src/PrestaShopBundle/Translation/KeysReference/README.md

```diff
@@ -0,0 +1,2 @@
+This directory aims to get all translation keys that need to be referenced, but not cearly visible in code.
```

Comment:
cearly? what does it means?

Reply:
That means translation tool need to fetch `trans()` method to retrieve and store translations (what a mess :smile: ): https://github.com/PrestaShop/TranslationToolsBundle/blob/master/Translation/Extractor/Visitor/TranslationNodeVisitor.php